added intro file

claeszombie opened the pull request:
intro file

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

claeszombie committed:
added intro file

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+Hello im claeszombie
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

claeszombie committed:
added favoritue path

claeszombie committed:
added

claeszombie committed:
new skill

codeschool-kiddo committed:
Added new rule about coffee

claeszombie committed:
edited

claeszombie committed:
Merge pull request #2 from claeszombie/new_rules

Too many people are spilling coffee

claeszombie committed:
new mr

anch 'deadly_skills'

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

claeszombie committed:
Hotfix

claeszombie committed:
merge hotfix

claeszombie committed:
hghgjsd

claeszombie committed:
mergar

anch 'release_branch_1.0'

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.
